evm_arithmetization/tests/basic_smart_contract.rs

```diff
@@ -198,7 +199,16 @@ fn test_basic_smart_contract() -> anyhow::Result<()> {
     };
 
     let mut timing = TimingTree::new("prove", log::Level::Debug);
-    let proof = prove::<F, C, D>(&all_stark, &config, inputs, &mut timing, None)?;
+    let max_cpu_len = 1 << 20;
+    let proof = prove::<F, C, D>(
+        &all_stark,
+        &config,
+        inputs,
+        max_cpu_len,
+        0,
+        &mut timing,
+        None,
+    )?;
```

Comment:
for all the integration tests, it's a bit weird to explicitely store in a var `max_cpu_len` but not the `segment_idx`. Maybe do it as well for the latter for clarity?